### ISSUE TITLE
Wallabag can't connect to RabbitMQ server

### ISSUE THREAD

**Issue by andyforceno** [wallabag/wallabag#3254]
I'm trying to import roughly 800 urls from Instapaper, but I'm running into an issue where Wallabag can't connect to rabbitmq-server (running on localhost). I'm not sure why. I have all of the relevant configuration options set (server address, user, password). I can connect to it, another web app I am running seems to have no problems connecting to it: onlyoffice-documentserver is able to open and convert documents, and afaik, that relies on rabbitmq. 

The import fails without error, it just stops somewhere around 100. I'm not sure if getting Wallabag connected to rabbitmq-server is going to solve the issue, but I'd like to try. Oddly, I was able to do this last night, without any errors or issues. Nothing, as far as I know, has changed on my system. I have, of course, tried restarting the relevant services, and even a reboot. 

The message on the import page says to check the RabbitMQ configuration. What should I be checking for?

**Comment by andyforceno** [comment]
Actually, the import may be crawling along. I refreshed the page on a whim, and it's up to 300 now.

Anyway, as for rabbitmq, here is the log file:
    ERROR REPORT==== 27-Jun-2017::22:11:10 ===
    connection <0.530.0>, channel 1 - soft error:
    {amqp_error,not_found,"no queue 'wallabag.import.pocket' in vhost '/'",
                'basic.get'}
    
    =INFO REPORT==== 27-Jun-2017::22:11:10 ===
    closing AMQP connection <0.530.0> (127.0.0.1:42436 -> 127.0.0.1:5672)
    
    =INFO REPORT==== 27-Jun-2017::22:41:31 ===
    accepting AMQP connection <0.731.0> (127.0.0.1:47302 -> 127.0.0.1:5672)
    
    =ERROR REPORT==== 27-Jun-2017::22:41:31 ===
    connection <0.731.0>, channel 1 - soft error:
    {amqp_error,not_found,"no queue 'wallabag.import.pocket' in vhost '/'",
                'basic.get'}
    
    =INFO REPORT==== 27-Jun-2017::22:41:31 ===
    closing AMQP connection <0.731.0> (127.0.0.1:47302 -> 127.0.0.1:5672)`

So, it is connecting, but there's an error.

**Comment by j0k3r** [comment]
To get the wallabag version, browse to your_wallabag_url/about

Could you do a `rabbitmqadmin list channels` and `rabbitmqadmin list queues` using the command line and show us the output?

**Comment by andyforceno** [comment]
Wallabag v2.2.3

`rabbitmqadmin list channels:`

    +---------------------------------------+-------+
    |                 name                  | user  |
    +---------------------------------------+-------+
    | 127.0.0.1:35726 -> 127.0.0.1:5672 (1) | guest |
    | 127.0.0.1:35726 -> 127.0.0.1:5672 (2) | guest |
    | 127.0.0.1:35726 -> 127.0.0.1:5672 (3) | guest |
    | 127.0.0.1:35728 -> 127.0.0.1:5672 (1) | guest |
    | 127.0.0.1:35728 -> 127.0.0.1:5672 (2) | guest |
    | 127.0.0.1:35730 -> 127.0.0.1:5672 (1) | guest |
    | 127.0.0.1:35730 -> 127.0.0.1:5672 (2) | guest |
    +---------------------------------------+-------+

`rabbitmqadmin list queues:`

     +--------------------------------+----------+
     |              name              | messages |
     +--------------------------------+----------+
     | amq.gen-abE5YuBlPvLqd9nYj6khYA | 0        |
     | ds.convertdead                 | 0        |
     | ds.convertresponse             | 0        |
     | ds.converttask                 | 0        |
     | wallabag.import.instapaper     | 0        |
     +--------------------------------+----------+

**Comment by j0k3r** [comment]
The _we can't connect to it_ is because you only have one queue defined instead of all queue for each import. We check for all queue to be activated, if one failed (with reason `NOT_FOUND - no queue 'wallabag.import.pocket' in vhost '/'`) we assume Rabbit is now well configured.

We should change the message to list available queue in Rabbit instead.

So basically right now, you Rabbit is fine for the instapaper import.
As far as I understand articles are imported but you noticed that error, right?

**Comment by andyforceno** [comment]
@j0k3r Yes, the articles imported successfully, I was just wondering if the warning message was a bug or not, given that rabbitmq seemed accessible. Your explanation cleared that up, though. I think a more verbose (or just more precise) warning would be helpful.
